pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -288,6 +295,13 @@ func (r *ReconcileComplianceScan) phasePendingHandler(instance *compv1alpha1.Com
 		return reconcile.Result{}, err
 	}
 
+	if _, hasTimeoutAnn := instance.Annotations[compv1alpha1.ComplianceScanTimeoutAnnotation]; hasTimeoutAnn {
```

Comment:
This is a nit, but I think it would read nicer if we could wrap this in a method like `NeedsRescan`, maybe we could even share the body of the methods and just pass the name of the annotation into the inner method.

Reply:
that's a good suggestion, I will use a method here